Modules/QR Reader/MITScannerMgr.m

```diff
@@ -86,7 +88,9 @@ - (void)setupCaptureSession
     self.deviceInput = [[AVCaptureDeviceInput alloc] initWithDevice:self.captureDevice error:&error];
     if( self.deviceInput == nil )
```

Comment:
@bskinner No worries! 

I made a change to check `authorizationStatusForMediaType` before the captureSession is setup, and tested to make sure it works as expected in various cases. 

Also, I realized that in building services I don't check for camera authorization at all, so made a similar change there as well.